index.js

```diff
@@ -62,9 +63,16 @@ function choo (opts) {
   // (str?, obj?) -> DOMNode
   function start () {
     _store.model(createLocationModel(opts))
-    const createSend = _store.start(opts)
+    const createSend = _store.start(xtend(opts, {
+      subscriptions: false,
+      noSubscriptions: true
```

Comment:
Is it a design omission that there are both `subscriptions` and `noSubscriptions` options? Seems to me these two options overlap! Basically, you can't set just `subscriptions` as it will make Barracks think that subscriptions have been called, which isn't the case.